standards/open-standards.md

```diff
@@ -33,7 +33,7 @@ order: 10
 
 ## Management: what you need to do
 
-* ???
+* MAY Include open standard compliance assessment in code reviews.
```

Comment:
Everywhere else we're not using these codewords outside of the 'Measure', but this seems fine